lib/providers/services/http_service.dart

```diff
@@ -179,6 +172,42 @@ class HttpService {
     }
   }
 
+  Future<List<MemberModel>> getMembersByIds(List<String> accountNames) async {
```

Comment:
Only real change in this file. 

Added a new function to search for multiple users by accountName